### ISSUE TITLE
V4.2.1 On groups for PAGES when someone joins I can not approve the member as box is hidden

### ISSUE THREAD

**Issue by PhpFoxJohnJr** [moxi9/phpfox#1082]
On my site there is actual two issue although one seems fixed on my computer but not on a friend computer.

First  when a user  joins a page a box will pop up and say
![notice](https://cloud.githubusercontent.com/assets/14913734/12852953/21d56684-cbf7-11e5-8fc9-7a1d8e62c3b2.png)
and the message is missing.  But when I did my temporary fix it seemed to fix it on my computer and the message showed up but not on my friends computer...very odd since the fix below has nothing to do with this area.

Anyways, the other issue is after the person join a notice is sent to the admin of the page.  When you click on Management for page you will see a pending membership link.  Click on that and profile image come up for the member pending membership.  I am able to click on the member and then in the left bottom corner a box will appear and say with selected.  When you click on it a the pop up box does not appear.  I found the temporary solution for me by going to bootstrap.css in the flavors at line 18105

.moderation_holder ul {
 left: 0px;
 border: 0px;
 background: #ffffff;
 color: #555555;
 top: -100px;
}

I added the top: -100px; as the box was there it was just below the screen. 

Don't know if it is just me or if everyone is having this problem.  I changed from the wisteria theme to the bootstap theme now but I did do changes to fix the wisteria theme.

**Comment by Data66PF** [comment]
duplicate #1063 for the notice block with no info in it.

**Comment by Data66PF** [comment]
One reason we need one bug per report. :) Testing on my 4.2.2 dev site and I see the other issue this is for. If I have a pending membership notification, and I click that notification, it takes me to the page just fine but there is no place that the pending notification shows on the page itself. This is non-intuitive and the user does not know where to find the pending user without some (1) showing up next to the gear where they are supposed to click to find the pending users. (1) being the number of pending users. We need to improve this to add a number or some indication at the gear where they need to approve users so the page admin knows where to click. This is important as sometimes, they might miss the notification at the top of the site so we do need to indicate someplace that there is a pending member (other than just the top site notification).
![pending group member](https://cloud.githubusercontent.com/assets/6908282/13056941/d1e3e338-d3e6-11e5-8392-2ca2c6bfb93c.png)

**Comment by PhpFoxJohnJr** [comment]
I totally agree...your are right.

**Comment by robert9592** [comment]
Fixed in v4.3.0

**Comment by PhpFoxJohnJr** [comment]
Thanks Robert...

Did you guys determine if this was a bug or not...as I listed the main bug which I guess was a duplicate and Data found another bug which you said you fixed but is the other think I fixed on my site a bug or not...see statement below I copied from above....

Anyways, the other issue is after the person join a notice is sent to the admin of the page. When you click on Management for page you will see a pending membership link. Click on that and profile image come up for the member pending membership. I am able to click on the member and then in the left bottom corner a box will appear and say with selected. When you click on it a the pop up box does not appear. I found the temporary solution for me by going to bootstrap.css in the flavors at line 18105

.moderation_holder ul {
 left: 0px;
 border: 0px;
 background: #ffffff;
 color: #555555;
 top: -100px;
 }

I added the top: -100px; as the box was there it was just below the screen. 

Don't know if it is just me or if everyone is having this problem. I changed from the wisteria theme to the bootstap theme now but I did do changes to fix the wisteria theme
